compiler/optimizations/remoteValueForwarding.cpp

```diff
@@ -70,6 +72,8 @@ void remoteValueForwarding() {
     Map<Symbol*, Vec<SymExpr*>*> defMap;
     Map<Symbol*, Vec<SymExpr*>*> useMap;
 
+    replaceRecordWrappedRefs();
```

Comment:
It's not obvious to me from the surrounding code. Is replaceRecordWrappedRefs going to be called for single-locale compilation as well? And this runs in parallel(), right?

I'd consider moving replaceRecordWrappedRefs into its own file or its own pass. Nonetheless I understand it is less disruptive to put it here.